docs/devguide/docs/faq.md

```diff
@@ -0,0 +1,35 @@
+# Frequently Asked Questions
+
+## Installation
+
+### <b>Predator is running, how do I access the UI?</b>
+
+The UI is accessible at http://$MACHINE_IP/ui where `$MACHINE_IP` is your local network address you used to install Predator.
+                        
+## Tests
+
+### <b>I run a test successfully but no report is created for the test</b>
```

Comment:
why not write what we wrote in the installation part?

Reply:
good point

Reply:
actually, i think it's too detailed there and this is a good sum up of the actual problem + solution. In the end it's the same solution but worded a tiny bit differently

Reply:
ok so maybe add link to the installation docker anchor